Ms 5 add change personal photo

DamianBednarek opened the pull request:
- Implemented functionality to add or update user picture
- Wrote tests and API documentation
- Integrated storage to work locally in development settings and in amazon s3 storage bucket in production settings

DamianBednarek committed:
MS-5 -add-change-personal-photo

Created views and logic for adding or updating profile picture for user

DamianBednarek committed:
MS-5-add-change-person-picture

Created tests for adding and updating picture

DamianBednarek committed:
MS-5-add-change-personal-picture

-created API documentation for adding and updating picture
- refactored imports

DamianBednarek committed:
Updated tests for adding picture

DamianBednarek committed:
Improved tests

DamianBednarek committed:
removed not necessary serializer class declaration

dmitry-sergeev commented on MusicHub/users/serializers.py:
```diff
@@ -82,3 +82,24 @@ class ResetPasswordEmailSerializer(serializers.Serializer):
+    def validate(self, attrs):
+        picture = attrs["profile_avatar"]
+        # picture.size is in bytes
+        if picture.size > 3000000:
```
magic number

dmitry-sergeev reviewed (commented):
the same issues as https://github.com/Henok-U/MusicHubNew/pull/29

DamianBednarek committed:
Merge branch 'dev' into MS-5-add-change-personal-photo

DamianBednarek committed:
MS-5-add-change-picture

- Fixed merge conflicts
- Refactored tests
- Moved picture validation to profile_service

danisidiatullin commented on MusicHub/users/profile_service.py:
```diff
@@ -1,5 +1,7 @@
 from rest_framework.serializers import ValidationError
 
+MAX_PICTURE_SIZE = 3000000  # value in bytes, max 3Mb
```
When you write numbers with more than 3 digits by hand you can use underscore `_` as separator for better readability:
```
>>> 12345678 == 12_345_678
True
```
